### DISCUSSION TITLE
higress 非k8s环境独立部署，配合nacos 2.1.2无法正常拉取nacos中已注册的服务。

### DISCUSSION THREAD

**Discussion by h31105** [alibaba/higress#470] [Q&A]
部署方式：
curl -fsSL https://higress.io/standalone/get-higress.sh | bash -s -- -c nacos://192.168.1.195:8848 --nacos-username=nacos --nacos-password=nacos -p admin
部署环境：
centos7
nacos v2.1.2 docker部署 net=host 模式

higress与nacos部署在同一台服务器上。

**Comment by johnlanni** [comment]
在这里跟进把 https://github.com/alibaba/higress/issues/471
